compiler-core/src/language_server/engine.rs

```diff
@@ -211,10 +211,12 @@ where
                     Some(this.completion_types(module))
                 }
 
-                Located::ModuleStatement(Definition::Import(_) | Definition::ModuleConstant(_)) => {
-                    None
+                Located::ModuleStatement(Definition::Import(_)) => {
```

Comment:
Unfortunately this isn't going to work most the time as you're not going to be inside an import statement when trying to complete an import, as the AST isn't updated until there has been a successful compilation. It might be if we had fault tolerant parsing though. I'm not sure how's best to approach this- do you have ideas?

Reply:
Yeah that's a fair point. My original plan was to at least get the plumbing working to have import completions in this issue and figure out how to generally apply it outside of import statements as part of #2757 

I guess for most cases outside of the import statement itself we'd probably actually be completing for function/consts in other modules and just adding the import as an additional text insert so it'd be a bit different.

Fault tolerant parsing would definitely make this more usable as is but I would need to read more about how that actually gets implemented in practice